### PR TITLE
TEC-1644: Progress Bar

### PR DESCRIPTION
**Pull request by ocaioaugusto**

### CODE REVIEW

File: src/components/ProgressBar/index.jsx [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+      &:after {
+        content: "";
+        display: block;
+        width: 10px;
+        height: 10px;
+        background-color: #fff;
+        border-radius: 50%;
+        border: 3px solid #ec720b;
+        position: absolute;
+        left: 0px;
```
Comment (antoniojunyor) [inline src/components/ProgressBar/index.jsx:74]: Replace `0px` to `0`

File: src/components/ProgressBar/index.jsx [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+      position: absolute;
+      right: -10px;
+      z-index: ${zIndexes.STRATOSPHERE};
+    }
+  }
+
+  ${({ active }) => active === 0 && css`
+    & span:first-child {
+      &:after {
+        content: "";
```
Comment (antoniojunyor) [inline src/components/ProgressBar/index.jsx:66]: Use single quote

File: src/components/ProgressBar/index.jsx [hunk 3]
```diff
@@ -0,0 +1,87 @@
+import styled, { css } from 'styled-components';
+import { sizes, zIndexes, colors } from '../../styles/variables';
+
+export const Bar = styled.span`
```
Comment (antoniojunyor) [inline src/components/ProgressBar/index.jsx:4]: Maybe you can have a specific file to the `<Bar />` component, and another file to the `<ProgressBarStyled />`

File: src/components/ProgressBar/index.jsx [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+  & span:nth-child(${({ active }) => active}) {
+    &:after {
+      content: "";
+      display: block;
+      width: 10px;
+      height: 10px;
+      background-color: #fff;
+      border-radius: 50%;
+      border: 3px solid #ec720b;
```
Comment (antoniojunyor) [inline src/components/ProgressBar/index.jsx:56]: This color on border should be on variables?

File: src/components/ProgressBar/index.jsx [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 6px;
+  margin-right: 0.2%;
+  opacity: 0.2;
+  position: relative;
+
+  &:nth-child(1) {
```
Comment (antoniojunyor) [inline src/components/ProgressBar/index.jsx:14]: Just a comment, you don't need the `&` ... just to you know.

File: src/components/ProgressBar/index.jsx [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+        border-radius: 50%;
+        border: 3px solid #ec720b;
+        position: absolute;
+        left: 0px;
+        z-index: ${zIndexes.STRATOSPHERE};
+      }
+    }
+  `}`;
+
+export const ProgressBarContainer = styled.div`
```
Comment (antoniojunyor) [inline src/components/ProgressBar/index.jsx:80]: Create a specific file to this component

File: src/styles/variables.js [hunk 7]
```diff
@@ -43,3 +48,12 @@ export const fontWeights = {
   REGULAR: '400',
   BOLD: '700',
 };
+
+export const zIndexes = {
```
Comment (antoniojunyor) [inline src/styles/variables.js:52]: NICE!

**Review by antoniojunyor — CHANGES_REQUESTED** [review]

**Review by antoniojunyor — APPROVED** [review]

**Review by antoniojunyor — APPROVED** [review]

**Comment by pserick** [comment]
Tenta colocar a foto do component no pr pra ficar melhor pra analisar! 🙏🏽

**Review by pserick — APPROVED** [review]

**Pull request merged**